frontend/src/modules/displays/ModeBanner.tsx

```diff
@@ -32,21 +43,41 @@ export const getModeText = (mode: VentilationMode | null): string => {
   }
 };
 
+/**
+ * @typedef Props
+ *
+ * Props Interface for the Mode Banner component
+ *
+ * @prop {string} bannerType banner type for deciding font color
```

Comment:
What are the allowed/expected values of `bannerType`? If we were to turn this into an enum, what would its values be?

Reply:
Allowed values of `enum` could be normal/screenSaver. Its used to differentiate color styling of banner